source/_components/alert.markdown

```diff
@@ -58,6 +58,11 @@ entity_id:
   description: The ID of the entity to watch.
   required: true
   type: string
+title:
+  description: >
+    A title to be used for the notification if the notifier supports it.
```

Comment:
We should mention that this can be a template.

Reply:
Done!